include/wx/regex.h

```diff
@@ -63,6 +63,8 @@ enum
     wxRE_NOTEOL = 64
 };
 
+static const wxString s_strMetaChars = wxT("\\^$.|?*+()[]{}");
```

Comment:
This shouldn't be in the header but in the source file.

And it shouldn't use `wxT()` (never use it in the new code).

Finally, I'd make it a `const char*` and use `strchr()`, but this is just me.